### ISSUE TITLE
[2.0] [Discussion] Silent Partner

### ISSUE THREAD

**Issue by jbaxleyiii** [NewSpring/Norma#233]
After a fantastic discussion driven by the talented @benwiley. I think it is high time to think through removing traces of Norma outside of the Norma file. Also, after Hashicorp dropped the bomb that is [Otto](https://ottoproject.io/) <-- I mean WOW, I've been inspired to keep Norma as lightweight on projects as possible. This will 100% be a breaking change.

##### Changes:

1. Norma shouldn't touch the package.json for interact with a project's NPM. This will make support for non node projects seamless and prevent Norma from being installed without desire via npm install.
2. Norma will use the `.norma` folder to store node_modules, added configuration, and local project caching.
3. Settings will still be stored in `.norma` along side other assets.

@benwiley this change will also make running Norma side by side with meteor possible as Meteor ignores .<folders> (to my best knowledge) and the Norma file will have no effect on Meteor.

@johnthepink @delianides @benwiley thoughts?

Also, so far Otto is beautiful. Seriously impressive and humbling

**Comment by johnthepink** [comment]
@jbaxleyiii I agree. I don't think Otto replaces Norma. I think they could work well together.

**Comment by benwiley** [comment]
@jbaxleyiii I think this is the right direction.  We can get the much desired "developer assistant" help without feeling as if we've added the complexities of a whole second platform.
